package.json

```diff
@@ -571,7 +571,7 @@
     },
     "scripts": {
         "vscode:prepublish": "npm run cleanReadme",
-        "build": "tsc && gulp webpack-prod",
+        "build": "tsc",
```

Comment:
Is this intentionally included?

Reply:
Yeah, the `npm run build` shouldn't run webpack. It wasn't actually changing the line `return await extension.activate(ctx, perfStats, true);` either (where the true is ignoreBundle) so it wasn't actually prepping it to be used with webpack.

Reply:
Can confirm that this matches our other extensions

Reply:
Yeah, agreed: https://github.com/microsoft/vscode-azureresourcegroups/issues/826